chrome/content/binding.xml

```diff
@@ -16,7 +16,7 @@
 
   <binding id="recommendation-urlbar" extends="chrome://browser/content/urlbarBindings.xml#urlbar">
     <handlers>
-      <handler event="keypress"><![CDATA[
+      <handler event="keypress" phase="capturing"><![CDATA[
```

Comment:
![burn it down](https://cloud.githubusercontent.com/assets/23885/13499170/641c126c-e11a-11e5-9f34-73bf6b3058d1.gif)